[2024-08-29T00:53:36.884+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-29T00:53:36.927+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: welcome_dag.print_random_quote scheduled__2024-08-27T23:00:00+00:00 [queued]>
[2024-08-29T00:53:36.955+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: welcome_dag.print_random_quote scheduled__2024-08-27T23:00:00+00:00 [queued]>
[2024-08-29T00:53:36.956+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2024-08-29T00:53:37.019+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): print_random_quote> on 2024-08-27 23:00:00+00:00
[2024-08-29T00:53:37.026+0000] {standard_task_runner.py:72} INFO - Started process 476 to run task
[2024-08-29T00:53:37.031+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'welcome_dag', 'print_random_quote', 'scheduled__2024-08-27T23:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dag.py', '--cfg-path', '/tmp/tmphg51u6eo']
[2024-08-29T00:53:37.034+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask print_random_quote
[2024-08-29T00:53:37.198+0000] {task_command.py:467} INFO - Running <TaskInstance: welcome_dag.print_random_quote scheduled__2024-08-27T23:00:00+00:00 [running]> on host ecb62bc16934
[2024-08-29T00:53:37.457+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='welcome_dag' AIRFLOW_CTX_TASK_ID='print_random_quote' AIRFLOW_CTX_EXECUTION_DATE='2024-08-27T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-27T23:00:00+00:00'
[2024-08-29T00:53:37.458+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-29T00:53:38.343+0000] {logging_mixin.py:190} INFO - Quote of the day: "Knowledge rests not upon truth alone, but upon error also."
[2024-08-29T00:53:38.344+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-29T00:53:38.365+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-29T00:53:38.366+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=welcome_dag, task_id=print_random_quote, run_id=scheduled__2024-08-27T23:00:00+00:00, execution_date=20240827T230000, start_date=20240829T005336, end_date=20240829T005338
[2024-08-29T00:53:38.453+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-29T00:53:38.484+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-29T00:53:38.486+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
